src/reference/asciidoc/amqp.adoc

```diff
@@ -4471,59 +4729,59 @@ Some properties are not exposed by the namespace; indicated by `N/A` for the att
 
 | (group)
 
-| This is only available when using the namespace.
+| This is available only when using the namespace.
 When specified, a bean of type `Collection<MessageListenerContainer>` is registered with this name, and the
 container for each `<listener/>` element is added to the collection.
-This allows, for example, starting/stopping the group of containers by iterating over the collection.
-If multiple `<listener-container/>` elements have the same group value, the containers in the collection is
+This allows, for example, starting and stopping the group of containers by iterating over the collection.
+If multiple `<listener-container/>` elements have the same group value, the containers in the collection form
 an aggregate of all containers so designated.
 
 a| image::images/tickmark.png[]
 a| image::images/tickmark.png[]
 
-| channelTransacted
+| `channelTransacted`
```

Comment:
These "literalizations" (is that a word :D ?) are not needed because the column is specified as containing literals...

`[cols="8l,16,3,3", options="header"]`

(the `8l`).

I am not sure it hurts anything, although in Spring Integration it did cause some overflows in the docbook PDF generation, so I have generally reverted them back over there.

But, since we're also getting rid of the dockbook PDF maybe its even more a non-issue.

Reply:
I missed that. Sorry. I'm glad you caught it.